lib/daru/index.rb

```diff
@@ -330,5 +342,12 @@ def values
     def inspect
       "Daru::MultiIndex:#{self.object_id} (levels: #{levels}\nlabels: #{labels})"
     end
+
+    def display input_indexes
+      # Remove levels not needed for display
+      return self if input_indexes[0].is_a? Range
+      remove_levels = input_indexes.length
+      drop_left_level remove_levels
```

Comment:
write this as `drop_left_level input_indexes.size`. No need for the extra variable.